scripts/apps/authoring/tests/authoring.spec.ts

```diff
@@ -124,7 +124,7 @@ describe('authoring', () => {
                 expect($scope.dirty).toBe(false);
                 expect(api.save).toHaveBeenCalled();
             })
-            .finally(done);
+            .finally.catch(done.fail).catch(done.fail).catch(done.fail).catch(done.fail).catch(done.fail).catch(done.fail).catch(done.fail).catch(done.fail).catch(done.fail).catch(done.fail).catch(done.fail).catch(done.fail).catch(done.fail).catch(done.fail).catch(done.fail).catch(done.fail).catch(done.fail).catch(done.fail).catch(done.fail).catch(done.fail).catch(done.fail).catch(done.fail).catch(done.fail).catch(done.fail).catch(done.fail).catch(done.fail).catch(done.fail).catch(done.fail).catch(done.fail).catch(done.fail).catch(done.fail).catch(done.fail).catch(done.fail).catch(done.fail).catch(done.fail).catch(done.fail).catch(done.fail).catch(done.fail).catch(done.fail).catch(done.fail)(done);
```

Comment:
:rofl: you shouldn't run lint with a "fix" option. We should probably drop it from `packgage.json` too.

Reply:
I know, this was just my editor using the linter to format the file on save... I had to disable it every time I opened that file 😅